src/createBaseForm.js

```diff
@@ -452,7 +452,7 @@ function createBaseForm(option = {}, mixins = []) {
           const name = field.name;
           if (options.force !== true && field.dirty === false) {
             if (field.errors) {
-              set(alreadyErrors, name, { errors: field.errors });
+              alreadyErrors[name] = { errors: field.errors };
```

Comment:
Developers may use `alreadyErrors.user.age`, so I think your PR will be a breaking change.

> The other solution maybe change the way validateFieldsAndScroll

I think this will work, and you can use `lodash.has` for we had already import this dependence.